motor_control/motor_control.h

```diff
@@ -0,0 +1,24 @@
+#ifndef motor_control_h
+#define motor_control_h
+
+#include <Wire.h>
+#include <Adafruit_MotorShield.h>
+#include "utility/Adafruit_MS_PWMServoDriver.h"
+
+class Mmotor {
```

Comment:
Can we name it as something else? Writing this name could easily cause typos that will be hard to catch

Reply:
Maybe RobotMotor?